ci/terraform/oidc/account-interventions-alerts.tf

```diff
@@ -32,12 +32,12 @@ resource "aws_cloudwatch_metric_alarm" "account_interventions_p1_cloudwatch_alar
 
 data "aws_cloudwatch_log_group" "auth_account_interventions_lambda_log_group" {
   depends_on = [module.account_interventions]
-  count      = var.environment == "production" || var.environment == "integration" ? 0 : 1
+  count      = var.environment == "production" || var.environment == "integration" ? 1 : 0
```

Comment:
I think for the log groups and metric filters, these should just be fully on in all environments (as per orch's code). I think worth cross referencing each of the things here against orch's ones above and aligning them

Reply:
Going to delete our alarms in this file as @dbes-gds pointed out that there are already alarms set up for every endpoint automatically for auth in endpoint-module/alerts.tf